plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py

```diff
@@ -107,6 +107,33 @@ def test_jobs_configured_via_config(hydra_sweep_runner: TSweepRunner) -> None:
         assert math.isclose(best_parameters["quadratic.y"], -1.0, abs_tol=1e-4)
 
 
+def test_jobs_configured_via_config_logscale(hydra_sweep_runner: TSweepRunner) -> None:
```

Comment:
thanks for adding the tests. 

there is a lot of duplication going on here, could you use [pytest params](https://docs.pytest.org/en/6.2.x/example/parametrize.html) to avoid all these duplication?

for example, it seems that `param` and the `overrides` are the only params that are different, you could set those two as pytest parameters to avoid copying the whole test. you can see an example in the same test file https://github.com/facebookresearch/hydra/blob/main/plugins/hydra_ax_sweeper/tests/test_ax_sweeper_plugin.py#L36-L52

same comment goes for `test_jobs_configured_via_cmd_logscale`

Reply:
Will fix.